stat/stat.go

```diff
@@ -1076,6 +1076,8 @@ func empiricalQuantile(p float64, x, weights []float64, sumWeights float64) floa
 // Skew computes the skewness of the sample data.
 // If weights is nil then all of the weights are 1. If weights is not nil, then
 // len(x) must equal len(weights).
+// Note that in case of weights summing to less than (or equal) to 1,
```

Comment:
Please remove the parentheses here and below.

I would prefer something like "`// When weights sum to 1 or less, a biased variance estimator should be used.`"

Reply:
That is indeed clearer, thanks!